data/en/census_household.json

```diff
@@ -3636,7 +3631,7 @@
                         "id": "employers-business",
                         "questions": [{
                             "id": "employers-business-question",
-                            "title": "What is the main activity of <em>{{answers.company_name}}</em>?",
+                            "title": "What is the main activity of <em>{{answers['business-name-answer']}}</em>?",
```

Comment:
Two issues here:

- the census schema wasn't defining which group instance to get the business name from
- the updated schema context builder wasn't forcing the instance values to be put in the correct position in their list. This wouldn't have been a problem if completing the survey linearly but if you jumped around it would fail